README.md

```diff
@@ -1,2444 +1,28 @@
-  - [Surge](#surge)
-- [Advanced Configuration](#advanced-configuration)
-- [Troubleshooting](#troubleshooting)
-  - [`npm start` doesn’t detect changes](#npm-start-doesnt-detect-changes)
-  - [`npm test` hangs on macOS Sierra](#npm-test-hangs-on-macos-sierra)
-  - [`npm run build` exits too early](#npm-run-build-exits-too-early)
-  - [`npm run build` fails on Heroku](#npm-run-build-fails-on-heroku)
-  - [`npm run build` fails to minify](#npm-run-build-fails-to-minify)
-  - [Moment.js locales are missing](#momentjs-locales-are-missing)
-- [Alternatives to Ejecting](#alternatives-to-ejecting)
-- [Something Missing?](#something-missing)
+## Learning Goals
+In this project you will learn to:
+- Build React components which recieve data through props
+- Build React container components which pass information to their subcomponents.
 
-## Updating to New Releases
+## Project Outline
 
-Create React App is divided into two packages:
+We have provided you an initial Application Skeleton generated with `create-react-app`.  In the `src/data` folder you will have a JSON file filled with timeline data for a specific user.  The application will have the following components:
 
-* `create-react-app` is a global command-line utility that you use to create new projects.
-* `react-scripts` is a development dependency in the generated projects (including this one).
+- `App` - This component is the traditional outer component of the React App. The App component will read in the data from the JSON file and pass the timeline events to the `Timeline` component which it will contain.
+- `Timeline` - This component will take a list of timeline events as a prop and render a list of `TimelineEvent` components.
+- `TimelineEvent` - This component will take an individual timeline event data and render it.  Each `TimelineEvent` will contain a person who posted the event, a status message and a timestamp.
+- `Timestamp` - This component will take timestamp data and display the number of hours since the post was created.  An example would be `6 hours ago`.
 
-You almost never need to update `create-react-app` itself: it delegates all the setup to `react-scripts`.
+Implement the `render` functions of each component.  An effective strategy could be to implement each component individually testing their output on the screen with hard-coded prop data before trying to integrate the entire app.
 
-When you run `create-react-app`, it always creates the project with the latest version of `react-scripts` so you’ll get all the new features and improvements in newly created apps automatically.
+Below is a sample solution:
 
-To update an existing project to a new version of `react-scripts`, [open the changelog](https://github.com/facebookincubator/create-react-app/blob/master/CHANGELOG.md), find the version you’re currently on (check `package.json` in this folder if you’re not sure), and apply the migration instructions for the newer versions.
+![Sample App](./images/example-feed.png)
```

Comment:
Now that the final styling is complete (check out the latest from `Hamled/impl`) we should update the image because the page looks significantly different.

Reply:
Yeah, this screenshot is also super difficult to read

Reply:
updated image